app.sample.json

```diff
@@ -0,0 +1,13 @@
+{
+    "expo": {
+        "extra": {
+            "IDPS": ["read.test"],
+            "LANGUAGE_CODE": "?"
+        },
+        "revisionId": "?",
+        "splash": {
+            "backgroundColor": "?",
+            "resizeMode": "?"
```

Comment:
In a comment on this pull request, you mentioned `"resizeMode": "contain", backgroundColor": "#ffffff"` so perhaps include these here as suggestions.